docs/builds/guides/frameworks/angular.md

```diff
@@ -33,6 +33,10 @@ All available versions are [listed on npm](https://www.npmjs.com/package/@ckedit
 
 ## Quick start
 
+<info-box>
+	CKEditor 5 packages do not include TypeScript typings yet. With strict mode enabled, you must take care of the typings. **Please bear in mind that projects generated using Angular CLI have strict mode enabled by default.** Otherwise, you might see the `Could not find a declaration file for module` error. For more information, please see [Strict mode project tips](#strict-mode-project-tips).
```

Comment:
```suggestion
	CKEditor 5 packages do not include TypeScript typings yet. With strict mode enabled, you must take care of the typings yourself. Otherwise, you might see the `Could not find a declaration file for module` error. **Please bear in mind that projects generated using Angular CLI have strict mode enabled by default.**
	
	For more information, please see [Strict mode project tips](#strict-mode-project-tips).
```